Index: alsa-ucm-conf/ucm2/HDA/HiFi-mic.conf
===================================================================
--- alsa-ucm-conf.orig/ucm2/HDA/HiFi-mic.conf
+++ alsa-ucm-conf/ucm2/HDA/HiFi-mic.conf
@@ -297,7 +297,7 @@ If.micsetup {
 		Define {
 			DeviceMicName "Mic"
 			DeviceMicComment "Stereo Microphone"
-			DeviceMicJack "Mic Jack"
+			DeviceMicJack "${var:MicJackControl}"
 			DeviceMicPriority 200
 		}